gc/base/Collector.hpp

```diff
@@ -196,18 +196,12 @@ class MM_Collector : public MM_BaseVirtual
 	 * @param env[in] The thread which performed the change in heap geometry 
 	 */
 	virtual void heapReconfigured(MM_EnvironmentBase *env, HeapReconfigReason reason, MM_MemorySubSpace *subspace, void *lowAddress, void *highAddress)
-	{
-		heapReconfigured(env); /* Required temporarily to not break dependency with heapReconfigure API changes */
-	};
-	
-	/* Required temporarily to not break dependency with heapReconfigure API changes */
-	virtual void heapReconfigured(MM_EnvironmentBase *env) 
 	{
 		/* ALl collectors except Scavenger & Realtime should have their own implementation */
 		MM_GCPolicy gc_policy = env->getExtensions()->configurationOptions._gcPolicy;
 		Assert_MM_true(gc_policy == OMR_GC_POLICY_METRONOME || gc_policy == OMR_GC_POLICY_GENCON);
```

Comment:
I'd remove this assert now. Any collector is free to override this in future.

Reply:
👍 Done